.circleci/continue_config.yml

```diff
@@ -26,20 +26,33 @@ parameters:
     type: boolean
     default: false
 
-jobs:
-  ci-base:
+executors:
+  python:
     parameters:
       python-version:
         type: string
-        default: "3.9"
-      extras:
+        default: "3.9.18"
+      resource-class:
         type: string
-        default: "none"
+        default: "small"
     docker:
       - image: cimg/python:<< parameters.python-version >>
-    resource_class: small
+    resource_class: << parameters.resource-class >>
```

Comment:
Is there a reason why this got changed as well?

Reply:
Thought to do some small refactor since this setup is repeated.